app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java

```diff
@@ -14,18 +14,15 @@
         PAUSED, LOOPING, DOWNLOADING, DOWNLOADED, NOT_DOWNLOADED, DOWNLOAD_FAILED
     }
 
-    @Value
     public abstract String getUuid();
 
     @Value.Default
     public String getPath() {
         return "";
     }
 
-    @Value
     public abstract String getUrl();
-
-    @Value
```

Comment:
They aren't permited to be used on methods...i tried here. :(
I can put a comment like `//abstract methods are immutables by default